core/lib/config/src/configs/eth_sender.rs

```diff
@@ -63,6 +64,15 @@ pub enum ProofLoadingMode {
     FriProofFromGcs,
 }
 
+#[repr(u16)]
+#[derive(Debug, Deserialize, Clone, Copy, PartialEq, Serialize, Default)]
+pub enum PubdataSendingMode {
+    #[default]
+    Calldata = 0,
+    OneBlob,
+    TwoBlobs,
+}
+
```

Comment:
Why is this a `#[repr(u16)]` `enum`? Can it be a regular `enum` with a single field for any number of blobs?

```rust
pub enum PubdataSendingMode {
   Calldata,
   Blobs(NonZeroU16),
}
```

Reply:
the u16 was left over from when it was being stored in the DB, will remove. i think given that we'll only support 1-2 blobs that we dont need to have it be that flexible. in the long run, the number of blobs will likely be dynamic